launcherctl/_app.py

```
@@ -14,21 +14,21 @@ def is_paused(self) -> bool:
-        launcherctl("pause-app", self.name)
+        _ = launcherctl("pause-app", self.name)
 
     def resume(self) -> None:
-        launcherctl("resume-app", self.name)
+        _ = launcherctl("resume-app", self.name)
 
 
 class API:
     def keys(self) -> list[str]:
-        return [x.decode("utf-8") for x in launcherctl("list-apps").splitlines()]
+        return [x for x in launcherctl("list-apps").splitlines()]
```

Ruff (0.15.11)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)